web/client/utils/openlayers/OlLocate.js

```diff
@@ -210,15 +237,37 @@ OlLocate.prototype.mapClick = function(evt) {
 };
 
 OlLocate.prototype._getDefaultStyles = function() {
-    return new Style({
-        image: new CircleStyle({
-            radius: 6,
-            fill: new Fill({color: 'rgba(42,147,238,0.7)'}),
-            stroke: new Stroke({color: 'rgba(19,106,236,1)', width: 2})
-        }),
+    const color = '#2A93EE';
+    const shape = 'navigation_arrow';
+    const circleAccuracyStyle = {
         fill: new Fill({color: 'rgba(19,106,236,0.15)'}),
         stroke: new Stroke({color: 'rgba(19,106,236,1)', width: 2})
-    });
+    };
+
+    return {
+        locate: new Style({
+            image: new CircleStyle({
+                radius: 6,
+                fill: new Fill({color}),
+                stroke: new Stroke({color: 'rgba(19,106,236,1)', width: 2})
+            }),
+            ...circleAccuracyStyle
+        }),
+        circleAccuracy: circleAccuracyStyle,
+        heading: {
+            iconLibrary: 'standard',
+            iconAnchor: [0.5, 0.5],
+            anchorXUnits: 'fraction',
+            anchorYUnits: 'fraction',
+            color,
+            fillColor: color,
+            opacity: 1,
+            size: 41,
+            fillOpacity: 1,
+            shape,
+            symbolUrl: `product/assets/symbols/${shape}.svg`
```

Comment:
Use directly the raw xml. it is small and can be included in the bundle without requiring URL.